css files/gethelpnow-style.css

```diff
@@ -1,15 +1,9 @@
-/* Created by Akanksha
-   Desc: Style for Get Help Now module and it's submodules
-   Use in the following php files
-   getHelpNow
-   getHelpNow2
-   PCSaves
-   OfficeOfVictimAdvocacy
-   OfficeOfCivilRightsAndDiversity
-   OfficeOfInspectorGeneral
-   twilioSMS
-   twilioCall
- */
+/* DOCUMENT INFORMATION
+    - Created by : Akanksha
+    - Desc: Style for Get Help Now module and it's submodules
```

Comment:
Please format the comments [like this](https://github.com/twbs/bootstrap/blob/v4-dev/dist/css/bootstrap.css#L1)